masterlist.yaml

```diff
@@ -188,6 +188,11 @@ common:
         text: 'Delete or deactivate %1%.'
       - lang: ru
         text: 'Не забудьте отключить или удалить %1%.'
+  - &redundantPluginX
+    type: warn
+    content:
+      - lang: en
+        text: 'This plugin is redundant when using %1% and should be deactivated or deleted.'
```

Comment:
A new message specific to the case where resources should be kept seems more appropriate to me than using `alreadyInX`.